spec/lib/zendesk_apps_tools/translate_spec.rb

```diff
@@ -101,6 +101,41 @@
     end
   end
 
+  describe '#write_json' do
+    let(:translate) { ZendeskAppsTools::Translate.new }
+
+    before do
+      allow(translate).to receive(:options) { { unattended: true } }
+    end
+
+    around(:example) do |example|
+      Dir.mktmpdir do |dir|
+        @test_file_path = "#{dir}/test.json"
+        example.run
+      end
+    end
+
+    it 'works for identical ascii' do
+      translate.create_file(@test_file_path, JSON.pretty_generate({ node: "test abc" }) + "\n", force: true)
+      expect { translate.write_json(@test_file_path, { node: "test abc" }) }.to output("   identical  #{@test_file_path}\n").to_stdout
```

Comment:
hm, we'd have to re-write this without the rpsec nicety to capture stdout as a StringIO then read it as a String to lstrip: http://stackoverflow.com/a/22777806/1911487

Since it adds extra code I'm happy not to bother